### PR TITLE
Initmodel

### PR DESCRIPTION
**Pull request by michellinh**
==> Title: Initmodel 

==> Redmine: https://edu-redmine.sun-asterisk.vn/issues/19259

==> Attachment File:

Screenshot from 2019-11-18 09-54-17

### CODE REVIEW

**Comment by michellinh** [comment]
ready

File: app/models/combo.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+class Combo < ApplicationRecord
+  belongs_to :restaurant
+  has_many :combodetails, dependent: :destroy
```
Comment (ngocvt-0484) [inline app/models/combo.rb]: để tên bảng combo_detail
order_detail hơn e ạ

**Comment by michellinh** [comment]
ready

File: app/models/user.rb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+class User < ApplicationRecord
```
Comment (manhbnt-0960) [inline app/models/user.rb]: model không ghi gì à e ?

**Comment by michellinh** [comment]
ready

File: app/models/order_detail.rb [hunk 3]
```diff
@@ -0,0 +1,4 @@
+class OrderDetail < ApplicationRecord
+  belongs_to :order
+  belongs_to :order_what, polymorphic: true
```
Comment (manhbnt-0960) [inline app/models/order_detail.rb]: dùng order_detailable thay order_what đi e

**Comment by michellinh** [comment]
ready

**Comment by michellinh** [comment]
ready

**Comment by michellinh** [comment]
ready

**Pull request merged**